server/models/Collective.js

```diff
@@ -876,18 +876,20 @@ function defineModel() {
 
   // Save image it if it returns 200
   Collective.prototype.checkAndUpdateImage = async function (image) {
-    try {
-      const response = await fetch(image);
-      if (response.status !== 200) {
-        throw new Error(`status=${response.status}`);
-      }
-      const body = await response.text();
-      if (body.length === 0) {
-        throw new Error(`length=0`);
+    if (process.env.OC_ENV !== 'e2e') {
```

Comment:
It's not only for E2E, unit tests are affected too

Reply:
Thanks for the review; I've corrected that. 👍🏽